modules/client-configuration/pages/client-upgrades-uyuni.adoc

```diff
@@ -0,0 +1,89 @@
+[[client-upgrades-uyuni]]
+= Upgrade Uyuni Clients
```

Comment:
You are talking about openSUSE Leap here only. Specifically to upgrade to Leap 15.2.

Remember SLE can also be managed with Uyuni.

Reply:
Thanks, we now have https://github.com/SUSE/spacewalk/issues/12793 --I just created it ;)